Libplanet.Net/Consensus/ConsensusContext.Event.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Libplanet.Action;
+using Libplanet.Net.Messages;
+
+namespace Libplanet.Net.Consensus
+{
+    public partial class ConsensusContext<T>
+        where T : IAction, new()
+    {
+        /// <inheritdoc cref="Context{T}.ExceptionOccurred"/>
+        internal event EventHandler<(long Height, int Round, Exception)>? ExceptionOccurred;
+
+        /// <inheritdoc cref="Context{T}.TimeoutOccurred"/>
+        internal event EventHandler<
+                ((long Height, int Round) Context, (Step Step, TimeSpan TimeSpan) Event)>?
+            TimeoutOccurred;
```

Comment:
The only additional "variable context" from `ConsensusContext` is `Height`. I think it'd be better to have

```
(long Height, (int Round, Step Step, TimeSpan TimeSpan) Event)>?
```

and additionally add `Round` for `Context` event handlers for overall consistency.

Reply:
https://github.com/planetarium/libplanet/blob/53cc604384598000490e77f65e036e5165e1f420/Libplanet.Net/Consensus/ConsensusContext.Event.cs#L13-L16
https://github.com/planetarium/libplanet/blob/53cc604384598000490e77f65e036e5165e1f420/Libplanet.Net/Consensus/ConsensusContext.Event.cs#L61-L69

Amended. I just realized that unwrapping tuple is a one way to do so...

https://github.com/planetarium/libplanet/pull/2383/files#diff-d5fcd6416d32a07c2ab500ee36b81596dfd297f498c1c4d691040141b5c1ebd3

Reply:
I think either way is fine as long as there is some semblance of consistency 😗